### ISSUE TITLE
[FEATURE] Is there any way to save the logits of the classification task?

### ISSUE THREAD

**Issue by yufu** [Megvii-BaseDetection/cvpods#75]
**Please use this template to submit your need.**

---

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by FateScript** [comment]
Please use template to specify your problem.
